src/app/pages/card-import/card-import.page.html

```diff
@@ -50,5 +51,10 @@
       </ion-button>
     </ion-col>
   </ion-row>
+    <ion-button expand="full" fill="solid" color="secondary" (click)="sendCard()">
```

Comment:
This is just like the user-details page. It may be useful to have a send button included with the update and delete buttons for an existing card (if id exists) and a only create function (if id does not exist). It's okay to make a case for your ideas.